UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -315,10 +315,10 @@
   gUefiCpuPkgTokenSpaceGuid.PcdCpuDisableMtrrProgramming|TRUE
   
   # Above 4G Decode
-!if $(ENABLE_ABOVE_4G_DECODE) == TRUE
-  gUefiPayloadPkgTokenSpaceGuid.PcdEnableAbove4GDecode|TRUE
+!if $(ABOVE_4G_DECODE) == TRUE
+  gUefiPayloadPkgTokenSpaceGuid.PcdAbove4GDecode|TRUE

```

Comment:
any reason this can't be reduced to `gUefiPayloadPkgTokenSpaceGuid.PcdAbove4GDecode|$(ABOVE_4G_DECODE)` ?

Reply:
That was the first attempt, and it'll throw a bunch of errors (I still have them if you want them?)

I guessed "PcdsFixedAtBuild" doesn't allow variables

Reply:
Seems to work now, I guess it was something else causing it not to work!

Reply:
I take it back...